.circleci/config.yml

```diff
@@ -92,14 +92,14 @@ jobs:
               cd ..
               git clone $AUTH_CLONE_URL
               cd aws-amplify-cypress-auth
-              yarn install
+              yarn install --frozen-lockfile
```

Comment:
I don't think this is needed. This is a test repo with UI code that we use to run our cypress tests

Reply:
Removed install, checkout needs to stay as we discussed.